map.js

```diff
@@ -90,8 +112,8 @@ function moveMap(event){
     var wStyle = document.getElementById("wrapper_Map").style;
     var wX = wStyle.width.slice(0,wStyle.width.length-2);
     var wY = wStyle.height.slice(0,wStyle.height.length-2);
```

Comment:
why rely on the style of the element and not just the element width and height?

Reply:
updated to just use wrapper.width/height.